cmd/kpod/version.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/urfave/cli"
+)
+
+//Overwritten at build time
+var (
+	gitCommit string
+	buildInfo string
+)
+
+//Function to get and print info for version command
```

Comment:
nit: Follow the comment guidelines here and elsewhere in the PR. 
For reference, see https://golang.org/doc/effective_go.html#commentary

So this could something like:
```
// versionCmd get and prints info for version command 
```